pydoc/setup/LINUX_INSTALL.md

```diff
@@ -0,0 +1,91 @@
+# Linux
+
+There is no Linux Chapel install, so the first two steps in the Linux Arkouda install are to install the Chapel dependencies followed by downloading and building Chapel. The Ubuntu and RHEL Chapel installations are different for installing Chapel dependencies, particularly regarding older RHEL distro versions such as CentOS 7. Specifically, the gcc compiler on RHEL distros such as CentOS 7 do not support building Chapel. Consequently, a newer version of the gcc compiler must be installed via the [devtoolset-9-gcc-c++](https://centos.pkgs.org/7/centos-sclo-rh-x86_64/devtoolset-9-gcc-c++-9.1.1-2.6.el7.x86_64.rpm.html) package. In addition, the CentOS [Software Collections packager](https://wiki.centos.org/AdditionalResources/Repositories/SCL) must be installed to enable the newer version of gcc to be leveraged for building Chapel.
+
+## Install Chapel (Ubuntu)
+
+```bash
+# Update Linux kernel and install Chapel dependencies
+sudo apt-get update
+sudo apt-get install gcc g++ m4 perl python3 python3-pip python3-venv python3-dev bash make mawk git pkg-config cmake llvm-12-dev llvm-12 llvm-12-tools clang-12 libclang-12-dev libclang-cpp12-dev libedit-dev
+
+# Download latest Chapel release, explode archive, and navigate to source root directory
+wget https://github.com/chapel-lang/chapel/releases/download/1.29.0/chapel-1.29.0.tar.gz
+tar xvf chapel-1.29.0.tar.gz
+cd chapel-1.29.0/
+
+# Set CHPL_HOME
+export CHPL_HOME=$PWD
+
+# Add chpl to PATH
+source $CHPL_HOME/util/setchplenv.bash
+
+# Set remaining env variables and execute make
+# It is recommended to add these variables to a ~/.chplconfig file to prevent having 
+# to export them again
+export CHPL_COMM=gasnet
+# If you're going to be running locally only, use CHPL_COMM=none instead to improve build time
+export CHPL_COMM_SUBSTRATE=smp
+export CHPL_TARGET_CPU=native
+export GASNET_QUIET=Y
+export CHPL_RT_OVERSUBSCRIBED=yes
+export CHPL_RE2=bundled
+export CHPL_LLVM=bundled
```

Comment:
we need to make sure to add `CHPL_GMP=bundled` for the bigint functionality

Reply:
Added